docs/guides/create-webshop.md

```diff
@@ -0,0 +1,208 @@
+
+## Prerequisites
+- General familiarity with [Orchard Core](https://docs.orchardcore.net/en/latest) in general, see [Tutorials](https://docs.orchardcore.net/en/latest/docs/resources/tutorials/).
+- An installed Orchard Core website
+
+## Creating a basic webshop
+
+### Step 1 — Install the Commerce module using NuGet
+Get the latest version of the [Orchard Core Commerce NuGet](https://www.nuget.org/packages/OrchardCore.Commerce) package. Should you vie for further tips regarding NuGet usage with Orchard Core, see the [Getting Started](https://docs.orchardcore.net/en/latest/docs/getting-started) page.
+
+### Step 2 — Enable necessary features
+Certain Orchard Core Commerce features, as well as some prerequisite stock Orchard Core features, need to be enabled for the necessary functionality. This can be done under *Configuration > Features*. These features include the following:
+
+- **Orchard Core Commerce - Core**: Registers core components of Commerce features.
+- **Orchard Core Commerce - Payment** Provides the basics for online payment.
+- **Orchard Core Commerce - Payment - Stripe**: Provides online payment implementation using Stripe.
+- **Orchard Core Commerce - Session Cart Storage**: Provides shopping cart functionality.
+- **Widgets**: Allows rendering widgets in zones.
+- **Layers**: Allows rendering widgets across pages based on conditions.
+- **Feeds**: Adds feeds capabilities.
+- **Lists**: Adds a content type that can be used as a container.
+
+![Enable features in bulk.](../assets/images/create-webshop/step-2/enable-features.png)
+_Using the bulk enable option makes this process swift._
+
+Optionally, enable a few other useful Commerce features as well to get even more out of your webshop. More on these later in the [corresponding section](#other-useful-commerce-features), but these include:
+
+- **Orchard Core Commerce - Inventory**: Provides basic inventory management.
+- **Orchard Core Commerce - Promotion**: Allows specifying discounts and promotions.
+- **Orchard Core Commerce - Tax**: Provides tax-related functionality.
+- **Orchard Core Commerce - Custom Tax Rates**: Enables locally maintaining tax rates.
+
+### Step 3 — Create Shopping Cart Widget
+Shopping without a shopping cart is a difficult task, and while it's not outside the realm of possibility, let's make our lives easier by creating one.
+For this, we're going to need to create Zones, Layers, and finally a Widget.
+
+- **Zones**:
+Navigate to _Design > Settings > Zones_ and define the zone where you want the shopping cart widget to be rendered. Standard Orchard Core zones include Header, Content, and Footer, so it might be best to start with those. To learn more about Zones, refer to their [documentation](https://docs.orchardcore.net/en/main/docs/reference/modules/Layers/#zones).
+
+![Zones.](../assets/images/create-webshop/step-3/zones.png)
+
+_Header, Content, Footer, any zone can be created here. Except for Ozone, that's illegal._
```

Comment:
I included this because it seems like the simplest way to get a Shopping Cart Widget displayed, and the zone setup process is related enough to have it included this way IMO.

I'll add that they need to be referenced by the Layout.